code/game/gamemodes/events/powerfruit.dm

```diff
@@ -32,7 +33,10 @@
 	add_avail(POWER_PER_FRUIT)
 
 	//spread - copypasta from spreading_growth.dm
-	if(prob(SPREAD_CHANCE))
+	var/chance = MIN_SPREAD_CHANCE + (powernet.avail / 1000) //two powercreeper plants raise chance by 1
+	chance = chance > MAX_SPREAD_CHANCE ? MAX_SPREAD_CHANCE : chance
+	message_admins("chance at [chance]%")
```

Comment:
leftover debug, just a heads up

Reply:
accidentally pushed that, thats for watching out